docker/opensource.md

```diff
@@ -1,6 +1,106 @@
 # Open Source
 {id: open-source}
 
+## TOML Kit
```

Comment:
If you look at the other page in the file you'll see that under each page title there is also an id that will become the url of the page.